### PR TITLE
Aet ui style mapping

### PR DESCRIPTION
**Pull request by sabeen**
rearrangement of mapping buttons.
The error log moves from mapping section to output section.

### CODE REVIEW

File: aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx [hunk 1]
```diff
@@ -37,19 +37,22 @@ class Mapping extends Component {
     return (
       <div className='section-body'>
         <div className='rules'>
-          { this.renderAddNewRuleButton() }
-
+          <h3 className='title-medium'>Mapping rules</h3>
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx]: `FormattedMessage` :see_no_evil:

File: aether-ui/aether/ui/assets/apps/pipeline/sections/Output.jsx [hunk 2]
```diff
@@ -5,9 +5,18 @@ class Output extends Component {
   render () {
     return (
       <div className='section-body'>
-        <code>
-          { JSON.stringify(this.props.selectedPipeline.output || [], 0, 2) }
-        </code>
+        <div className='pipeline-errors'>
+          <h3 className='title-medium'>Mapping Errors</h3>
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/pipeline/sections/Output.jsx]: `FormattedMessage` :see_no_evil:

File: aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx (hunk 1 above)
Comment (sabeen) [reply aether-ui/aether/ui/assets/apps/pipeline/sections/Mapping.jsx]: ok, and also for all the other headlines. got it. ;-)

File: aether-ui/aether/ui/assets/apps/pipeline/sections/Output.jsx [hunk 3]
```diff
@@ -5,9 +5,18 @@ class Output extends Component {
...
+          <code>
+            { JSON.stringify(this.props.selectedPipeline.mapping_errors || [], 0, 2) }
+          </code>
+        </div>
+        <div className='pipeline-data'>
+          <h3 className='title-medium'>Output Data</h3>
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/pipeline/sections/Output.jsx]: `FormattedMessage` :see_no_evil:

**Review by obdulia-losantos — CHANGES_REQUESTED** [review]
:fr: :gb: :de: :es: and so on...

**Review by obdulia-losantos — APPROVED** [review]

**Pull request merged**